1/styles/scss/_media-tablet.scss

```diff
@@ -0,0 +1,118 @@
+@media screen and (max-width: 992px) {
```

Comment:
![image](https://user-images.githubusercontent.com/84564899/119957669-f45ccc80-bfaa-11eb-8a00-710ec077df29.png) 
missed indent
we have tablet width up from 768px

Reply:
done